tests/actions/fetchOrdersAction.test.js

```diff
@@ -0,0 +1,57 @@
+import fetchOrders from '../../app/js/actions/fetchOrder';
+
+import {
+  FETCH_ORDERS_LOADING,
+  FETCH_ORDERS_SUCCESS
+} from '../../app/js/actions/actionTypes';
+describe('fetchOrders action', () => {
```

Comment:
since this is a test, you do not need to import real instances to use in the test, it is always advisable to mock if possible, it'd be better if you just used the string-literals here directly instead of importing your code.

Reply:
I believe the main purpose of declaring action types as constants and exporting them, is because of the need to mitigate human typographical error. Importing a const isn't necessarily mocking the actions.
it's just to ensure that no typos are being made when testing the actions.

Reply:
regarding importing real instances.
we import real `components`, `actions` and `reducers` to test them, I don't see anything wrong in importing real `actionTypes`.